recipe/meta.yaml

```diff
@@ -96,7 +96,6 @@ test:
 
 app:
   entry: orange-canvas
-  summary: data visualization and data analysis tool
```

Comment:
It's needed for Anaconda Navigator.

Reply:
Ignore the linter check

Reply:
Scheiße...is there a recipe directory to suppress the linter error?

Reply:
Inside `extra`:
```
extra:
  skip-lints:
  - duplicate_key_in_meta_yaml
```

Reply:
I was looking for this, thanks!